examples/voice_agent/README.md

```diff
@@ -122,6 +122,8 @@ Most LLMs from HuggingFace are supported. A few examples are:
 - [meta-llama/Llama-3.1-8B-Instruct](https://huggingface.co/meta-llama/Llama-3.1-8B-Instruct)
 - [nvidia/Llama-3.1-Nemotron-Nano-8B-v1](https://huggingface.co/nvidia/Llama-3.1-Nemotron-Nano-8B-v1) 
 - [nvidia/Nemotron-Mini-4B-Instruct](https://huggingface.co/nvidia/Nemotron-Mini-4B-Instruct)
+- [nvidia/NVIDIA-Nemotron-Nano-9B-v2](https://huggingface.co/nvidia/NVIDIA-Nemotron-Nano-9B-v2)
+    - Please see `server/server_config_vllm.yaml` for how to use this model with vllm server.
```

Comment:
I think this should be also added to `## Start the server` section for less of confusion.
Users need to replace the yaml file with this vllm config.